README.md

```diff
@@ -16,15 +16,35 @@ See most of the available options at the [Demo Page](http://marekdedic.github.io
 * All major desktop browsers and versions as well as mobile browsers on Android, iOS and Windows Phone.
 * IE8 is NOT supported
 
+### Install
+```sh
+// YARN
+yarn add jquery imagelightbox
+
+// NPM
+npm install jquery imagelightbox
+```
+
+After that include imagelightbox.css and imagelightbox(.min).js to your page.
```

Comment:
The CSS is now also available minified

Reply:
Also, it would be good to tell people the exact path - mostly because they should use the files from the `dist` directory